website/docs/docs/build/packages.md

```diff
@@ -56,7 +56,10 @@ The default [`packages-install-path`](/reference/project-configs/packages-instal
 You can specify a package using one of the following methods, depending on where your package is stored.
 
 ### Hub packages (recommended)
-[dbt Hub](https://hub.getdbt.com) is a registry for dbt packages. Packages that are listed on dbt Hub can be installed like so:
+
```

Comment:
remoe the callout as i think it should be its own sentence here introducing this section. plus it was rendering weird bc of the spacing. let me know if yo have strong opinions though!
![Screenshot 2024-03-21 at 10 23 16](https://github.com/dbt-labs/docs.getdbt.com/assets/89008547/c28c2d6d-75f9-4615-a57e-75db01e3687c)

Reply:
update now looks like this:
![Screenshot 2024-03-21 at 11 22 13](https://github.com/dbt-labs/docs.getdbt.com/assets/89008547/c0a5bed6-5e61-4113-97c3-474f294faf7c)